tests/subsys/suit/flash_sink/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+&mram1x {
+	erase-block-size = < 0x400 >;
```

Comment:
Not going to request changes but might be an idea to have one of these files and then just `#include "../..."` from the rest